measured.go

```diff
@@ -140,16 +140,21 @@ func Listener(l net.Listener, interval time.Duration) *MeasuredListener {
 func New() *Measured {
 	return &Measured{
 		// to avoid blocking when busily reporting stats
-		chStat: make(chan Stat, 10),
-		chStop: make(chan struct{}),
+		chStat:  make(chan Stat, 10),
```

Comment:
Why 10?
We've seen in high-load situations temporary errors submitting metrics. Could it be favored by this buffer being too shallow?